File: 문자열처리/5586/5586.cpp
#include <bits/stdc++.h>
using namespace std;

string s;

int main(void) {
    cin >> s;

    int JOI_num = 0;
    int IOI_num = 0;

    for (int i = 0; i < s.size() - 2; i++) {
        string temp = s.substr(i, 3);
        if (temp == "JOI") {
            JOI_num++;
        }

        if (temp == "IOI") {
            IOI_num++;
        }
    }

    cout << JOI_num << '\n';
    cout << IOI_num << '\n';
    return 0;
}

File: 문자열처리/5586/5586.py
str = input()

JOI_num = 0
IOI_num = 0
for i in range(len(str)-2):
    if(str[i:i+3] == 'JOI'):
        JOI_num += 1
    if(str[i:i+3] == 'IOI'):
        IOI_num += 1

print(JOI_num)
print(IOI_num)
